package.json

```diff
@@ -13,6 +13,7 @@
     "lint:tests": "tslint \"./tests/**/*.ts{,x}\" -c tslint.json --fix",
     "lint:templates": "standard \"./static/show-me/**/*.js\"",
     "lint": "npm-run-all \"lint:*\"",
+    "generate-test-report": "jest --config=jest.json --json > report.json",
```

Comment:
Can we call this `test:report`, since we already have `test:ci` and `test:coverage`?